backend/app/services/sms_service.py

```
@@ -0,0 +1,289 @@
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)